### DISCUSSION TITLE
edit label barcode rekam medis

### DISCUSSION THREAD

**Discussion by bektigalan** [mas-elkhanza/SIMRS-Khanza#97] [Q&A] [answered]
halo admin simrs khanza, saya ingin bertanya bagaimana cara mengubah label rekam medis? di menu berikut, file apa yang harus diganti? terimakasih
![image](https://user-images.githubusercontent.com/7407859/108160505-c9491e00-711b-11eb-9cd9-d39114b6eff5.png)

**Comment by mas-elkhanza** [comment, marked as answer]
edit pakai netbeans gan

**Reply by bektigalan** [reply]
siap, makasih mas 🙏
